starter-code/src/viking.js

```diff
@@ -1,11 +1,63 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+  this.health = health;
+  this.strength = strength;
+
+  Soldier.prototype.attack = function(){
+    return this.strength;
+  }
+
+  Soldier.prototype.receiveDamage = function(damage){
+    this.health -= damage;
+ 
 
 // Viking
-function Viking() {}
+function Viking(name, health, strength) {
+   
+  Soldier.call(this, health, strength);
+  this.name = name 
+
+}
+Viking.prototype = Object.create(Soldier.prototype);
+	
+Viking.prototype.constructor = Viking;
+
+Viking.prototype.attack = function (){
+  return this.strength;
+};
+
+Viking.prototype.receiveDamage = function(damage){
+  this.health -= damage;
+  if (this.health > 0){
+    var text = " has received " + damage + "points of damage";
```

Comment:
Lee lo que te está diciendo el test:
Espera: **"Harald has received 50 points of damage"**
Le estás devolviendo: **"Harald has received 50points of damage"**
¿Ves la diferencia?